web/config.py

```diff
@@ -22,6 +22,14 @@
 if sys.path[0] != root:
     sys.path.insert(0, root)
 
+# The number of connections to keep open inside the connection pool.
+# A SQLITE_CONNECTION_POOL_SIZE setting of 0 indicates no limit.
+CONFIG_DATABASE_CONNECTION_POOL_SIZE = 5
+# The number of connections to allow in connection pool "overflow".
```

Comment:
\# The number of connections allowed to overflow beyond the connection pool size.